crates/bpe-openai/src/lib.rs

```diff
@@ -86,6 +86,15 @@ impl Tokenizer {
             .sum()
     }
 
+    pub fn count_till_limit(&self, text: &str, token_limit: usize) -> Option<usize> {
```

Comment:
some comment for the function?
It needs at least some explanation what the return value is...

Reply:
(maybe also comments for the other pub functions)

Reply:
Done in https://github.com/github/rust-gems/pull/35/commits/c28e428f76bf3605e08e8ae251f46d7c72f8a4d2